site/content/docs/user/ingress.md

```diff
@@ -25,8 +25,40 @@ by the ingress controller `nodeSelector`.
 
 The following ingress controllers are known to work:
 
+ - [Contour](#contour)
  - [Ingress NGINX](#ingress-nginx)
 
+### Contour
+
+Create a kind cluster with `extraPortMappings` which will expose ports 80 & 443 from your cluster to your local host.
+
+{{< codeFromInline lang="bash" >}}
```

Comment:
This part is similar to the ingress nginx portmappings? Can we merge them
You'll also want to use node labels for making it work in multi node clusters.

Reply:
The configs are similar but Contour doesn't require those extra mappings which is why I split the configs apart. That said, having them for Contour won't break it, just not needed to get it running with Kind. 

How would this work with multi-node clusters in kind? You can't have additional ports map to 80/443?

Reply:
What I meant is, in the multi-node case, those node-labels and the deployments patches are required to ensure that the controller gets scheduled correctly to the node which has the ports exposed.

For example I just tried running contour ingress against a cluster with some additional nodes and it doesn't work properly.

```
cat <<EOF | kind create cluster --config=-
kind: Cluster
apiVersion: kind.x-k8s.io/v1alpha4
nodes:
- role: control-plane
  extraPortMappings:
  - containerPort: 80
    hostPort: 80
    protocol: TCP
  - containerPort: 443
    hostPort: 443
    protocol: TCP
- role: control-plane
- role: worker
- role: worker
EOF
```

Reply:
Ahhh ok, I'm following now. =) Let me fix this up.